sourcegraph/sourcegraph
# [SG-35011] - Accessibility: No description for branch pills

## Description
Add screen reader specific context to pills to indicate what they refer to.

## Problem description

<img width="202" alt="Screenshot 2022-05-05 at 21 13 46@2x" src="https://user-images.githubusercontent.com/19534377/167008855-2bce82a2-6f7c-457f-9e76-ff023c83f2b9.png">
these pills here are just read aloud by the screen reader without any context. They should likely get some sort of aria label.


## Expected behavior
It's clear to the listener what the two words and the arrow refer to.

## Ref
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/35011)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-35011)

## Test Plan
1. Navigate to batch changes list page
2. Select a batch change
3. Ensure that: using the screen reader, the branch pills and arrow are announced with more context as to what they refer to (Branch)

## App preview:

- [Web](https://sg-web-contractors-sg-35011.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-evmgdrzped.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
